### PR TITLE
comment

### PR DESCRIPTION
**Pull request by ethanlchristensen**

### CODE REVIEW

**Comment by nulzo** [comment]
@ethanlchristensen why are there conflicts

**Review by nulzo — CHANGES_REQUESTED** [review]
fix conflicts

**Comment by ethanlchristensen** [comment]
Sure thing

**Comment by ethanlchristensen** [comment]
@nulzo kindly fixed conflicts

File: BOT/_slash/marcus_says.py [hunk 1]
```diff
@@ -13,6 +13,9 @@ def __init__(self, tree: discord.app_commands.CommandTree, guild: str, args=None
         Description: Constructor.
         """
 
+        # args is passed in from slash_master. slash_master recieved args
```
Comment (Mr-Oregano) [inline BOT/_slash/marcus_says.py:16]: Args isn't capitalized! ☝️ 🤓

**Review by Mr-Oregano — DISMISSED** [review]

File: BOT/_slash/marcus_says.py (hunk 1 above)
Comment (ethanlchristensen) [reply BOT/_slash/marcus_says.py:16]: It shouldn't be @Mr-Oregano

**Review by nulzo — APPROVED** [review]

**Pull request merged**